float32 robot_x
float32 robot_y
float32 child_x
float32 child_y
float32 child1_x
float32 child1_y
float32 child2_x
float32 child2_y
float32 child3_x
float32 child3_y
float32 child4_x
float32 child4_y
float32 child5_x
float32 child5_y
float32 child6_x
float32 child6_y
float32 child7_x
float32 child7_y
float32 child8_x
float32 child8_y
float32 child9_x
float32 child9_y
